### DISCUSSION TITLE
Fixed NbExplorer

### DISCUSSION THREAD

**Discussion by WaliNeslIt** [btcpayserver/btcpayserver#5897] [Technical Support]
Do I need to manually develop the database for nbExplorer or can be done through auto migrate or need to install some other db engine instead of PostgreSQL

**Comment by dennisreimann** [comment]
It should work automatically and necessary migrations are applied when running NBXplorer. This goes for running it alongside BTCPay Server as well as the standalone version.

**Reply by WaliNeslIt** [reply]
actually
![Reply](https://github.com/btcpayserver/btcpayserver/assets/162506304/19d7c61f-137c-4acb-917d-f64d70607b59)
I did this change in configuration the btcpayserver db is been created while nbxplorer not created automatically

**Reply by WaliNeslIt** [reply]
NBXplorer not automatically created. also I didn't found the DBSet's for npxplorer
